edit/buildozer.go

```diff
@@ -74,19 +74,32 @@ type CmdEnvironment struct {
 
 // The cmdXXX functions implement the various commands.
 
-func cmdAdd(env CmdEnvironment) (*build.File, error) {
+func addImpl(env CmdEnvironment, concat bool) (*build.File, error) {
 	attr := env.Args[0]
 	for _, val := range env.Args[1:] {
-		if IsIntList(attr) {
-			AddValueToListAttribute(env.Rule, attr, env.Pkg, &build.LiteralExpr{Token: val}, &env.Vars)
-			continue
+		if concat {
+			expr := &build.LiteralExpr{Token: val}
+			ConcatListValueToListAttribute(env.Rule, attr, env.Pkg, expr, &env.Vars)
```

Comment:
use continue to avoid else blocks when possible.

So here, continue, then the else wrapper goes away